main.js

```diff
@@ -151,12 +163,14 @@ module.exports.shuttingDown = shuttingDown;
 
 const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));
 for (const file of commandFiles) {
+	/** @type {{data: import('@discordjs/builders').SlashCommandBuilder}} */
 	const command = require(`./commands/${file}`);
 	bot.commands.set(command.data.name, command);
 }
 
 const eventFiles = fs.readdirSync('./events').filter(file => file.endsWith('.js'));
 for (const file of eventFiles) {
+	/** @type {{name: String, once: Boolean, execute: function(...any)}} */
```

Comment:
Should be using primitive types instead.
```suggestion
	/** @type {{name: string, once: boolean, execute: function(...any)}} */
```